### PR TITLE
Endevor readme update

### PR DESCRIPTION
**Pull request by zeibura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,25 @@
...
 
+## Integration with Zowe Explorer and Explorer for Endevor
+
+We recommend that you use HLASM Language Support with [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe). Zowe Explorer enables you to open your High Level Assembler programs stored on mainframe data sets directly in VS Code. You can also integrate HLASM Language Support with [Explorer for Endevor](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.explorer-for-endevor) to retrieve your HLASM source code from Endevor locations and automatically retrieve dependencies. 
+
+HLASM Language Support, Zowe Explorer and Explorer for Endevor are all available as part of the [Code4z](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack) package.
+
+<a href="https://www.openmainframeproject.org/all-projects/zowe/conformance"><img alt="This extension is Zowe v2 conformant" src="https://artwork.openmainframeproject.org/other/zowe-conformant/zowev2/explorer/color/zowe-conformant-zowev2-explorer-color.png" width=20% height=20% /></a>
+
 ## Prerequisites
```
Comment (slavek-kucera) [inline README.md:24]: Should this be 3rd level header?
```suggestion
### Prerequisites
```

File: README.md [hunk 2]
```diff
@@ -7,15 +7,25 @@
...
 
-- There are no client or server-side prerequisites for HLASM Language Support.
+To enable automatic dependency retrieval from Endevor, install the following:
+- Endevor REST API version 2.16 or higher (PTF LU09053).
+- Explorer for Endevor version 1.70 or higher.
```
Comment (slavek-kucera) [inline README.md:28]:
```suggestion
- Explorer for Endevor version 1.7.0 or higher.
```

File: README.md [hunk 3]
```diff
@@ -7,15 +7,25 @@
...
 
 HLASM Language Support is also part of [Code4z](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack), an all-round package that offers a modern experience for mainframe application developers, including extensions for language support, data editing, testing, and source code management. For an interactive overview of Code4z, see the [Code4z Developer Cockpit](https://mainframe.broadcom.com/code4z-developer-cockpit).
 
+## Integration with Zowe Explorer and Explorer for Endevor
+
+We recommend that you use HLASM Language Support with [Zowe Explorer](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe). Zowe Explorer enables you to open your High Level Assembler programs stored on mainframe data sets directly in VS Code. You can also integrate HLASM Language Support with [Explorer for Endevor](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.explorer-for-endevor) to retrieve your HLASM source code from Endevor locations and automatically retrieve dependencies.
```
Comment (slavek-kucera) [inline README.md:18]: The integration with E4E is better/more advanced, would it make sense to mention it first?

File: README.md (hunk 1 above)
Comment (zeibura) [reply README.md:24]: No? Prerequisites is a L2 header.

Comment (slavek-kucera) [reply README.md:24]: ok

**Review by slavek-kucera — APPROVED** [review]
👍

File: README.md (hunk 3 above)
Comment (zeibura) [reply README.md]: Fixed now

**Pull request merged**